lib-ng/scanners/base.js

```diff
@@ -64,26 +58,33 @@ var BaseScanner = function (beaconMapper, beaconRegionMapper, beaconRegionMonito
         knownBeaconService.refreshBeacons([self.networkId]);
     };
 
+    this.isOldTiVersion = function () {
+        var version = Ti.version.split(".");
+        if (version[0] < 5) { // Version < 5
+            return true;
+        }
+        return (version[0] === 5 && version[1] === 0); // Version 5.0.*
+    };
+
     this.handleiOSLocationPermissions = function () {
         // Handle iOS
-        var version = Ti.version.split(".");
         var permissionType = Ti.Geolocation.AUTHORIZATION_ALWAYS;
-        if (isOldTiVersion()) { // Version 5.0.*
+        if (self.isOldTiVersion()) { // Version 5.0.*
```

Comment:
how did work with `this` work before? Was this change just in precaution if the function is used somewhere else?